cmd/tpmk/sshclient.go

```diff
@@ -63,6 +65,7 @@ need to be speficied in the command in the typical format:
 	flags.StringVarP(&opt.crtPassword, "crt-password", "n", "", "TPM NV index password")
 	flags.StringVarP(&opt.crtFormat, "crt-format", "f", "openssh", "Format of the client cert")
 	flags.BoolVarP(&opt.insecure, "insecure", "k", false, "Accept any host key")
+	flags.BoolVarP(&opt.insecure, "insecure", "P", "", "Socks5 proxy string")
```

Comment:
This should be setting `proxySocks5`, not `insecure`, and it should be a `StringVarP`.

Reply:
done